readme.md

```diff
@@ -1,14 +1,95 @@
 # Inexor
 
-Inexor is a fork of the quake-like FPS Cube 2: Sauerbraten.
-It adds a lot of functionality and tries to stay open for improvements and suggestions.
-The task is to be more flexible and create an environment where developement is easy and fast and creativity can prosper.
+Inexor is a fork of the open-source First-Person-Shooter Cube 2: Sauerbraten, a fast-paced shooting game featuring an ingame map editor.  
+In contrast to Sauerbraten, Inexor adds a lot of functionality and strives to stay open to improvements and suggestions.  
+The goal of this project is to be more flexible and create an environment where development is easy, fast and where creativity can prosper.
 
-You will find more information in our wiki.
+## How is Inexor organized?
 
+We are a non-hierarchical organisation. This means we are simply a group of people with different ideas working together without a leader making all the decisions. Any one of us is free to work on the particular things he/she wants to.
+For this organisation to work properly we rely on good communication. We are on IRC and Mumble pretty much everyday. Every so often we organise official Mumble meetings to discuss our roadmap and strategies.
+
+To contribute to the project and merge your changes into the master branch you need to convince enough members that the changes are an improvement.
+If you keep communicating what you are doing, take into account feedback and tips from others as well as giving insights into your decision making process, it probably won't be an issue (as long as it doesn't collide with our roadmap, of course).
+
+(Btw: We're open for new people!)
+
+## Where are we headed?
+
+Our goal is to make the game as moddable and developer-friendly as possible.
+Even though we might have refactored most of the code at some point, Inexor should always feel like Sauerbraten gameplay wise.
+
+A popular stance among the Sauerbraten community is "that's impossible", and this is what we want to prove wrong. Our answer to remarks like "things are best like they are" is: standing still means falling behind.
+
+
+###### Different features are afoot giving a general but no inevitably defined idea of where we're going next:
```

Comment:
Seems a bit much. h6 doesn't even exists.
And that's a really weird title anyways.